Add implications multiple src and invalid elements (#23085)
mdn/content: files/en-us/web/css/@font-face/index.md

  - : Defines a multiplier for glyph outlines and metrics associated with this font. This makes it easier to harmonize the designs of various fonts when rendered at the same font size.
- {{cssxref("@font-face/src", "src")}}

  - : Specifies font resources.

    A comma-separated list representing the resource fallback order, where each resource is specified using `url()` or `local()`.

>     The first resource in the list that loads successfully will be used, and subsequent items dropped/ignored.

Reviewer: ```suggestion
    The first resource in the list that loads successfully will be used and subsequent items are ignored.
```

    If multiple `src` descriptors are set, only the last declared rule that is able to load a resource is applied.

>     > **Note:** Items that the browser considers invalid are dropped.

Reviewer: Maybe we can continue with 'ignored'?

```suggestion
    > **Note:** Items that the browser considers invalid are ignored.
```

Author: Yes, sounds better.

>     > **Note:** Items that the browser considers invalid are dropped.

Reviewer: Maybe we can continue with 'ignored'?

```suggestion
    > **Note:** Items that the browser considers invalid are ignored.
```

    > Some browsers will ignore the whole descriptor if any item is invalid, even if only one item is invalid.
    > This may affect design of your fallbacks.

>     The `url()` can be followed by `format()` and `tech()`, like this:

Reviewer: Sorry I missed the notification on this one, I had a look over the pages and agree we should move the content out into the sub-doc like you suggest. 

>Was this intentional?

No, it doesn't look like it from the git history, it was added ad-hoc to the `@font-face` page, so a cleanup here sounds good. It doesn't look like I'll get to it before NY either, but I'm happy to tackle it next week if that sounds OK.

>     The `url()` can be followed by `format()` and `tech()`, like this:

Reviewer 2: Yeah, looks like something we'll be able to tackle in the new year.

>     The `url()` can be followed by `format()` and `tech()`, like this:

Author: Great. I'll leave that "bigger fix" to you to you guys then @dipikabh @bsmth

If the `local()` function is provided, specifying a font name to look for on the user's device, and if the {{Glossary("user agent")}} finds a match, that local font is used. Otherwise, the font resource specified using the `url()` function is downloaded and used.

> Browsers attempt to load resources in their list declaration order, so usually `local()` should be written before `url()`. Also, `local()` is not just a helper for `url()`, they are equal and both are optional, a rule block containing only one or more `local()` without `url()` is possible.

Reviewer: >they are equal and both are optional

I don't understand the implication of 'they are equal' 🤔

Author: I don't either - that bit of text predates me. I'm going to remove it though as unhelpful
```suggestion
Browsers attempt to load resources in their list declaration order, so usually `local()` should be written before `url()`. Both functions are optional, so a rule block containing only one or more `local()` without `url()` is possible.
```

> Further, if a more specific `format()` or `tech()` is desired, these should be specified _before_ the bare version, which would otherwise be tried first.

Reviewer: ```suggestion
If a more specific font with `format()` or `tech()` values is used, these should be listed _before_ versions that don't have these values as the less-specific variant would otherwise be tried and used first.
```

Author: ^^^ Yes, better. Slight variation used:

```suggestion
If a more specific fonts with `format()` or `tech()` values are desired, these should be listed _before_ versions that don't have these values, as the less-specific variant would otherwise be tried and used first.
```
